Examples/Nearest_Neighbour_Distance.ipynb

```diff
@@ -31,7 +31,9 @@
    },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    cat_subset = catalog['01deg_jra55v140_iaf_cycle4']
</pre><p>Already run in the cell above.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/363/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>